addons/crm/tests/test_crm_lead_convert_mass.py

```diff
@@ -166,7 +166,7 @@ def test_mass_convert_performances(self):
         test_leads = self._create_leads_batch(count=50, user_ids=[False])
         user_ids = self.assign_users.ids
 
-        with self.assertQueryCount(user_sales_manager=1041):  # 1038 generally on runbot - crm only: 1029 // 1040/41 about once / day on runbot (22h build ?)
+        with self.assertQueryCount(user_sales_manager=1144):  # crm only: 1004
```

Comment:
Is it still happening ? Still unsure why digest has an impact on this test ...

Reply:
Yes I have pushed the code without and I still got the same result for all test, except for "test_work_entries_generation_perf" for which I have removed the change.